src/commands/sources/query.ts

```diff
@@ -93,6 +93,7 @@ export class QuerySourcesCommand extends Command {
     const results = await service.queryAll({
       sources,
       query: flags.query,
+      timeoutMs: parseInt(process.env.QUERY_TIMEOUT as string) || 10000,
```

Comment:
Or we can retrieve the timeout value from the `flags` (similar to how we retrieve the `loglevel` value)? This allows a CLI user to set the timeout via the command line.

Reply:
Yes, done!